xarray_extras/kernels/csv.py

```diff
@@ -48,10 +48,10 @@ def to_csv(
 
     encoding = kwargs.pop("encoding", "utf-8")
     header = kwargs.pop("header", True)
-    line_terminator = kwargs.pop("line_terminator", os.linesep)
+    lineterminator = kwargs.pop("lineterminator", os.linesep)
```

Comment:
This bumps the minimum pandas version supported to 1.5.0. Which I'm OK with, but it needs to be documented and reflected in CI and in the pypi package.

Reply:
I'd need some hints where to update what, please.

Reply:
where it mentions pandas 1.1:
https://github.com/search?q=repo%3Acrusaderky%2Fxarray_extras%201.1&type=code

Reply:
I am a bit lost with the https://github.com/crusaderky/xarray_extras/blob/d661db851b772f20af9e6b88e856650720cfed5d/doc/whats-new.rst#L27, but I guess this isn't anyway something I could/should edit (?).